### DISCUSSION TITLE
ru/tut-making-platformer

### DISCUSSION THREAD

**Discussion by giscus** [ct-js/docs.ctjs.rocks#117] [Pages' discussions]
# ru/tut-making-platformer

Старый туториал! Здесь такие старые скриншоты и указания, что пылесос с пылью не справится. Вы можете помочь ct.js, переведя эту статью с английского (/tutorials/making-games-pl...

https://docs.ctjs.rocks/ru/tut-making-platformer.html

**Comment by jkeks** [comment]
Классная статья, пробую сделать свою игру на основе этой для практики.. очень хотелось бы видеть подобные туториалы для других жанров игр
